android/app/src/main/kotlin/coop/polypoly/polypod/polyNav/PolyNav.kt

```diff
@@ -18,6 +21,7 @@ open class PolyNav(
 ) {
     private val registeredActions = HashSet<String>()
     private val fsPrefix = "polypod://"
```

Comment:
We should *really* make this an integral part of the polyPod. Maybe in the new Meta instance variable. Again, outside the scope, but I just missed when it happened.